src/ploomber/cloud/api.py

```diff
@@ -170,12 +171,25 @@ def run_detail_print(run_id):
     tasks = out['tasks']
     run = out['run']
 
+    # pipeline has four status
     if run['status'] == 'created':
         click.echo('Run created...')
-    elif tasks:
+    elif run['status'] == 'aborted':
+        # both failed and aborted pipeline contain tasks
+        click.echo('Pipeline aborted...')
+        click.echo(Table.from_dicts(tasks))
+    elif run['status'] == 'failed':
+        click.echo('Pipeline failed...')
         click.echo(Table.from_dicts(tasks))
     else:
-        click.echo('Pipeline up-to-date, no tasks scheduled for this run.')
+        click.echo('Pipeline finished...')
+        if tasks:
```

Comment:
We can take this a level up and remove 183/180:
elif tasks:
  click.echo(Table.from_dicts(tasks))

Reply:
maybe for completeness also include  the `click.echo(Table.from_dicts(tasks))` in the finished case.

I'd also suggest explicitly doing `== 'finished'` and leave the else statement for an unknown case:

```
else:
   click.echo('Unknown status: {status}')
   click.echo(Table.from_dicts(tasks))
```